package.json

```diff
@@ -45,7 +45,7 @@
     "prebuild:watch": "yarn run clean",
     "build:watch": "tsc -w --strictNullChecks false",
     "prebuild": "yarn run clean",
-    "test": "NODE_ENV=test PORT=4243 jest",
+    "test": "NODE_ENV=test PORT=4243 node --trace-warnings node_modules/.bin/jest",
```

Comment:
This gives us the following trace in dev:

![Screenshot from 2024-03-12 15-53-47](https://github.com/Unleash/unleash/assets/455064/32f5f700-75b0-451a-aa09-2679e0133d95)